core/src/main/java/com/bumble/appyx/core/node/ParentNodeView.kt

```diff
@@ -4,7 +4,7 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import com.bumble.appyx.core.plugin.NodeAware
 
-abstract class ParentNodeView<Routing : Any> : NodeView, NodeAware<ParentNode<Routing>> {
+abstract class ParentNodeView<Routing : Any> : ParentNodeViewBase<Routing> {
```

Comment:
Can't say I'm a fan of the `Abstract` prefix but I don't have other options
```suggestion
abstract class AbstractParentNodeView<Routing : Any> : ParentNodeView<Routing> {
```